packages/fast-usdc/src/cli/cli.js

```diff
@@ -0,0 +1,160 @@
+import '@endo/init/legacy.js';
+import { Command } from 'commander';
+import { existsSync, mkdirSync, readFileSync } from 'fs';
+import { fileURLToPath } from 'url';
+import { dirname, resolve } from 'path';
+import { homedir } from 'os';
+import {
+  readFile as readAsync,
+  writeFile as writeAsync,
+} from 'node:fs/promises';
+import configLib from './config.js';
+import transferLib from './transfer.js';
+import { makeFile } from '../util/file.js';
+
+const packageJson = JSON.parse(
+  readFileSync(
+    resolve(dirname(fileURLToPath(import.meta.url)), '../../package.json'),
+    'utf8',
+  ),
+);
+
+const defaultHome = homedir();
+
+export const initProgram = (
+  configHelpers = configLib,
+  transferHelpers = transferLib,
+  readFile = readAsync,
+  writeFile = writeAsync,
+  mkdir = mkdirSync,
+  exists = existsSync,
+) => {
+  const program = new Command();
+
+  program
+    .name('fast-usdc')
+    .description('CLI to interact with Fast USDC liquidity pool')
+    .version(packageJson.version)
+    .option(
+      '--home <path>',
+      `Home directory to use for config`,
+      `${defaultHome}/.fast-usdc/`,
+    );
+
+  const config = program.command('config').description('Manage config');
+
+  const configFilename = 'config.json';
+  const getConfigPath = () => {
+    const { home: configDir } = program.opts();
+    return configDir + configFilename;
+  };
+
+  const makeConfigFile = () =>
+    makeFile(getConfigPath(), readFile, writeFile, mkdir, exists);
```

Comment:
most of the code doesn't seem to need write access.

Consider grabbing [makeFileRW and makeFileRd](https://github.com/Agoric/agoric-3-proposals/blob/4f553e0e084ec7a8203b7b5735d9ec0dfe2a57a7/packages/synthetic-chain/src/lib/webAsset.js) from synthetic-chain. make a RW and then call `.readOnly()`; only pass the RW to the things that need it.

Reply:
we can do that later. not critical.

Reply:
Ack